README.md

```diff
@@ -3,21 +3,40 @@ What?
 A lightweight Python package exposing a subset of OpenCL and SYCL
 functionalities.
 
+Requirements
+============
+- Install Conda
+- Install Intel oneAPI
+    - Set environment variable `ONEAPI_ROOT`
+        - Windows: `C:\Program Files (x86)\Intel\oneAPI\`
+        - Linux: `/opt/intel/oneapi`
+- Install OpenCL HD graphics drivers
 
-How to install?
-===
-1. Install oneAPI and OpenCL HD graphics drivers.
-2. Install conda or miniconda (you can use the conda that comes with oneAPI).
-3. [Optional] Create and activate a conda environment:
+Building and Install Conda Package
+==================================
+1. Create and activate conda build environment
+```bash
+conda create -n build-env conda-build
+conda activate build-env
+```
+2. Build conda package
+```bash
+conda build conda-recipe
+```
+On Windows to cope with long file names:
+```cmd
+conda build --croot=C:/tmp conda-recipe
+```
+3. Install conda package
+```bash
+conda install pydppl
+```
 
-    ```bash
-    conda env create -n dppl-env -f scripts/environment.yml`
-    conda activate dppl-env
-    ```
-4. Run `./scripts/build_for_conda.sh`
+Examples
+========
+See examples in folder `examples`.
 
-Examples:
-===
-   Run create_sycl_queues.py under examples.
-
-   `python examples/create_sycl_queues.py`
+Run examples:
+```bash
```

Comment:
Mention we need to source call "%ONEAPI_ROOT%compiler\latest\env\vars.bat"

Reply:
See `Using PyDPPL`